src/app/modules/spcp/spcp.errors.ts

```diff
@@ -96,3 +96,24 @@ export class MissingJwtError extends ApplicationError {
     super(message)
   }
 }
+
+/**
+ * SPCP form missing e-service ID.
+ */
+export class SpcpNoESrvcIdError extends ApplicationError {
```

Comment:
i didn't do this because `MyInfoNoESrvcIdError` has another callsite hwich is not related to this (and is `myInfo` specific). however, if this is desired, we can still keep this error and create a new sum type that is either `{Spcp | MyInfo}...Error`. 

i think you're correct and this serves to make code more readable so i'll go with teh above change!

Reply:
you can just use `FormAuthNoEsrvcIdError`, I don't think it's necessary to keep the MyInfo one separate

Reply:
sure, this sounds good and i'll make the changes accordingly! on that note, there's a `MyInfoAuthTypeError` along with `AuthTypeMismatchError` (in spcp) - i think i'll combine these also into `AuthTypeMismatchError` and shift it out into public-forms!